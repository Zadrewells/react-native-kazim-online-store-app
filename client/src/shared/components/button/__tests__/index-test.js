import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {
	Button
} from 'src/shared/components';

it('renders correctly', () => {
	const tree = renderer.create(
		<Button />
	).toJSON();

	expect(tree).toMatchSnapshot();
});
