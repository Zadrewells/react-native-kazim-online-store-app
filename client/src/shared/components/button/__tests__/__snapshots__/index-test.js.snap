// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<View
  accessible={true}
  buttonPadding="8px 8px 8px 8px"
  default={false}
  height="auto"
  icon={Object {}}
  onPress={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  outline={false}
  primary={true}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5197ff",
      "borderColor": "black",
      "borderStyle": "solid",
      "borderWidth": 0,
      "display": "flex",
      "flexDirection": "row",
      "height": "auto",
      "justifyContent": "center",
      "marginBottom": 0,
      "marginLeft": 0,
      "marginRight": 0,
      "marginTop": 0,
      "paddingBottom": 8,
      "paddingLeft": 8,
      "paddingRight": 8,
      "paddingTop": 8,
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
      "width": "auto",
    }
  }
  text="button primary"
  width="auto"
>
  <Text
    dark={false}
    default={true}
    fontSize={16}
    margin="0px 0px 5px 0px"
    primary={false}
    style={
      Array [
        Object {
          "alignSelf": "center",
          "color": "#ffffff",
          "fontFamily": "quicksand-bold",
          "fontSize": 16,
          "marginBottom": 5,
          "marginLeft": 0,
          "marginRight": 0,
          "marginTop": 0,
          "textAlign": "center",
          "textAlignVertical": "center",
        },
      ]
    }
    text="button primary"
  >
    button primary
  </Text>
  <View
    style={
      Array [
        Object {
          "alignItems": "flex-start",
          "display": "flex",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "marginRight": 0,
          },
        ]
      }
      tag={false}
    />
  </View>
</View>
`;
